### PR TITLE
Boosted test coverage.

### PR DESCRIPTION
**Pull request by jessetilro**
Created new test suites and wrote additional test cases for exisiting ones in order to bring the test coverage to an acceptable level. 

All static errors have been fixed or ignored. All packages except for the main package have a line coverage of 100%. Three out of the five packages have a branch coverage of 100%. See the Cobertura report for more details.

Resolves issue #114.

### CODE REVIEW

**Comment by dalderliesten** [comment]
Ran the tests, looks good to me.

File: src/test/java/nl/tudelft/pixelperfect/GameSmokeTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+   * little functionality to test here, just integration with the jMonkeyEngine.
+   * 
+   * Added the nice threading in order to be able to kill the application.
+   * 
+   * @throws InterruptedException
+   *           Since we're sleeping in this test.
+   * @throws AWTException
+   *           Since we're using the Robot.
+   */
+  @Ignore
```
Comment (dalderliesten) [inline src/test/java/nl/tudelft/pixelperfect/GameSmokeTest.java]: As a side note, this test suite is ignored until Jesse and I can figure out problems with our smoke tests.

**Comment by dmalarev** [comment]
I ran the tests, they ran perfectly

**Comment by wouterz** [comment]
LGTM

**Pull request merged**